pkg/lib/cockpit-components-install-dialog.css

```diff
@@ -0,0 +1,38 @@
+.package-list-ct {
```

Comment:
The commit log looks like you got a phone call when starting to type it? :-) ("lib/cockpit-components-install-dialog: New ").

Reply:
That was on purpose and how we did it back in the days. :-)  What else do you expect for a completely new module?